nats.go

```diff
@@ -2757,6 +2758,9 @@ func checkAuthError(e string) error {
 	if strings.HasPrefix(e, AUTHENTICATION_EXPIRED_ERR) {
 		return ErrAuthExpired
 	}
+	if strings.HasPrefix(e, `User Authentication Revoked`) {
```

Comment:
Make this a constant and all lower case. We convert the content of error string to lower (see line 2773)

Reply:
(add after `AUTHENTICATION_EXPIRED_ERR` const)